### PR TITLE
Add 0x DAO

### PR DESCRIPTION
**Pull request by cryptosabago**

### CODE REVIEW

File: protocols/0x/index.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "cname": "0x",
```
Comment (Rashmi-278) [inline protocols/0x/index.json:2]: Hey sandra, could you please change the cname to 0xgov ? 
It would be better if it's more than three characters

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]
cname and path needs to be changed to 0xgov

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]
Hey sandra, the changes look good ! 
There a few more additions needed. 
0xgov seems to have discourse and snapshot for their governance
can you please fill in the details of it too ? 

according to this https://docs.boardroom.info/adding-your-project/adding-your-project/protocol-information

here are links 
https://snapshot.org/#/0xgov.eth/
https://gov.0x.org/

**Comment by cryptosabago** [comment]
> https://gov.0x.org/

@Rashmi-278 
I searched high and low for this discourse link, I thought it would be under the domain: `discourse.org\SOMEDAO`. So I figured that they had nothing with discourse.
How does one find info about a DAO in general? 
I struggled in particular with the discourse, and the overview in general. 
Thanks

**Comment by Rashmi-278** [comment]
> @Rashmi-278 I searched high and low for this discourse link, I thought it would be under the domain: `discourse.org\SOMEDAO`. So I figured that they had nothing with discourse. How does one find info about a DAO in general? I struggled in particular with the discourse, and the overview in general. Thanks

Hey , the info is usually on the main website , if we don't find it we track their social media for links or updates and in general google for em.

Can you update the details of discourse links in PR ?

**Comment by Rashmi-278** [comment]
And for category we visit the discourse forun and click on the governance category then in URL look for the number

for this it's https://gov.0x.org/c/governance/13

**Comment by cryptosabago** [comment]
@Rashmi-278 Thanks again for the help. Did I miss anything this time?

**Review by Rashmi-278 — DISMISSED** [review]

**Comment by Rashmi-278** [comment]
> @Rashmi-278 Thanks again for the help. Did I miss anything this time?

Hey ! One last thing, we recently updated the docx so the discourse links format is changed.

```
discourseForum: {
  url: "",
  categoryId: "" 
}
```

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**